cmd/crane/cmd/serve.go

```diff
@@ -89,7 +96,11 @@ Contents are only stored in memory, and when the process exits, pushed data is l
 			return nil
 		},
 	}
-	cmd.Flags().BoolVar(&disk, "blobs-to-disk", false, "Store blobs on disk")
+	cmd.Flags().StringVarP(&disk, "blobs-to-disk", "", "", "Store blobs on disk")
+	// allow --blobs-to-disk to work without a value
+	cmd.Flags().Lookup("blobs-to-disk").NoOptDefVal = " "
```

Comment:
```suggestion
	cmd.Flags().Lookup("blobs-to-disk").NoOptDefVal = "[[[UNSET]]]"
```

(optional)

The thinking was that if someone might do `--disk=' '` (especially if they're doing something like `--disk=$DISK` and get unexpected behavior, and this makes it a bit harder to stumble into.

Reply:
hmm makes sense didn't think about that. fixing now.

Reply:
now that I think about it more, I can just deprecate --blobs-to-disk by printing a warning but still let it override the address.

Reply:
this is not necessary anymore.